packages/http-client/src/index.ts

```diff
@@ -6,7 +6,7 @@ import * as ifm from './interfaces'
 import * as net from 'net'
 import * as pm from './proxy'
 import * as tunnel from 'tunnel'
-import {ProxyAgent} from 'undici'
+import type ProxyAgent from 'undici/types/proxy-agent'
```

Comment:
is this line doing anything? I expect it should be shadowed below and so could be removed

Reply:
It's the return type of the method

Reply:
oh yeah. I see it used outside the diff now. nevermind :smile:  sorry for the distraction